support/index.html

```diff
@@ -86,6 +114,22 @@ <h2>Core Contributors</h2>
           the Open Source, Open Data and humanitarian mapping communities.
           </p>
         </div>
+        <div class="col-xs-10">
+            <ul class="experience">
+               <li>core</li>
+               <li>committer</li>
+               <li>support</li>
+            </ul>
+            <ul class="experience">
+               <li>development</li>
+               <li>coding</li>
+               <li>documentation</li>
+               <li>outreach</li>
+               <li><a href="https://www.astuntechnology.com/training/">training</a></li>
+               <li><a href="https://www.astuntechnology.com/support/">service</a></li>
+               <li><a href="https://www.astuntechnology.com/cloud-products/">product</a></li>
```

Comment:
@ianturton I tried to guess based on your website, please double check the above links.